src/utils/proxy/alternative-paths-for.mjs

```diff
@@ -0,0 +1,33 @@
+import isFunction from './is-function.mjs'
+
+// Used as an optimization to avoid dual lookups for missing assets
+const assetExtensionRegExp = /\.(html?|png|jpg|js|css|svg|gif|ico|woff|woff2)$/
+
+/**
+ * For an URL get alternative static file paths
```

Comment:
Can you add a bit more detail about what "alternative paths" means in this context?